content/en/device-support/_index.md

```diff
@@ -307,7 +307,108 @@ Passkeys created in **macOS** can be used on:
     </tr>
   </table>
   <details>
-    <summary>Advanced Capabilities</summary>
+    <summary><strong>Native Apps</strong></summary>
+    <div id="device-support-table" class="table-responsive">
+      <table class="table table-striped mt-0">
+        <thead>
+          <tr>
+            <td>Method</td>
+            <td class="text-center fw-bold">
+              <a href="/docs/reference/android/">Android</a>
+            </td>
+            <td class="text-center fw-bold">Chrome OS</td>
+            <td class="text-center fw-bold">
+              <a href="/docs/reference/ios/">iOS/iPad OS</a>
+            </td>
+            <td class="text-center fw-bold">
+              <a href="/docs/reference/macos/">macOS</a>
+            </td>
+            <td class="text-center fw-bold">Ubuntu</td>
+            <td class="text-center fw-bold">
+              <a href="/docs/reference/windows/">Windows</a>
+            </td>
+          </tr>
+          <tr class="align-middle">
+            <td class="fw-bold">
+                Native Platform APIs
+            </td>
+            <td class="text-center">
+              <i class="bi bi-check-circle-fill text-success fs-4"></i>
+            </td>
+            <td class="text-center">
+              <span class="fs-6 text-muted">n/a</span>
+            </td>
+            <td class="text-center">
+              <i class="bi bi-check-circle-fill text-success fs-4"></i>
+            </td>
+            <td class="text-center">
+              <i class="bi bi-check-circle-fill text-success fs-4"></i>
+            </td>
+            <td class="text-center">
+              <i class="bi bi-x-circle-fill text-danger fs-4"></i>
+              <br />
+              <span class="fs-6 text-muted">Not Supported</span>
+            </td>
+            <td class="text-center">
+              <i class="bi bi-check-circle-fill text-success fs-4"></i>
+            </td>
+          </tr>
+          <tr class="align-middle">
+            <td class="fw-bold">
+              System WebView
+            </td>
+            <td class="text-center">
+              <i class="bi bi-check-circle-fill text-success fs-4"></i>
+              <br />
+              <span class="fs-6 text-muted">Chrome<br>Custom Tabs</span>
+            </td>
+            <td class="text-center"><span class="fs-6 text-muted">n/a</span></td>
+            <td class="text-center">
+              <i class="bi bi-check-circle-fill text-success fs-4"></i>
+              <br />
+              <span class="fs-6 text-muted">AS<br>WebAuthentication<br>Session</span>
+            </td>
```

Comment:
I'd like to suggest use of the `<wbr>` element here in an effort to 1) make it easier to try and keep the columns narrow, and 2) make it easier to double-click any part of the word and have "ASWebAuthenticationSelection" selected (i.e. so that it's treated like a single word instead of three words):

```suggestion
            <td class="text-center" style="width: 135px">
              <i class="bi bi-check-circle-fill text-success fs-4"></i>
              <br />
              <span class="fs-6 text-muted">ASWebAuthentication<wbr>Session</span>
            </td>
```

The difference between this...

![2023-12-19 15-42-27 2023-12-19 15_42_57](https://github.com/passkeydeveloper/passkeys.dev/assets/5166470/5f15148a-90e7-41ef-80db-41a067d8656a)

...and this:

![2023-12-19 15-41-46 2023-12-19 15_42_16](https://github.com/passkeydeveloper/passkeys.dev/assets/5166470/02f93504-8699-4c8a-96ed-9a6712a1f236)

And my gut tells me it's better to keep "AS" attached to at least "WebAuthentication" for sake of searches for "ASWebAuthentication" being more likely to bring up Apple's `ASWebAuthenticationSession` docs. As opposed to a user visually seeing "AS WebAuthentication", searching for that, and getting nonsense because a search engine thinks it's two words.

Reply:
Alternatively, you could wrap the name of each of these platform APIs/capabilities in a link so it's easy to find the relevant docs:

```suggestion
            <td class="text-center" style="width: 135px">
              <i class="bi bi-check-circle-fill text-success fs-4"></i>
              <br />
              <span class="fs-6 text-muted">
                <a href="https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession" target="_blank" rel="noopener">
                  AS<wbr>WebAuthentication<br>Session
                </a>
              </span>
            </td>
```

To get something that maintains the narrower column while visually trying to present it as one long word:

![Screenshot 2023-12-19 at 3 46 54 PM](https://github.com/passkeydeveloper/passkeys.dev/assets/5166470/5e9c44af-3a1f-421b-aa4b-8e170a5ad6e6)

Reply:
will add terms and links in a future update